src/ResourceManager/Storage/AzureRM.Storage.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '5.0.2'
+ModuleVersion = '5.0.3'
```

Comment:
I think you'll probably want to bump the version of profile to the current version on the gallery (5.1.0) - line 54

Reply:
done...